R/render.r

```diff
@@ -422,7 +422,102 @@ data_deps <- function(pkg, depth) {
       )
     )
   )
-  deps <- bslib::bs_theme_dependencies(bs_theme)
+
+  # map secondary to component-active-bg (thus also dropdown-link-active-bg)
+  # unless a value was set by the user
+  if (is.null(pkg$meta[["template"]]$bslib$`component-active-bg`)) {
+    bs_theme <- bslib::bs_add_variables(
+      bs_theme,
+      "component-active-bg" = as.character(
```

Comment:
this is to force inheritance on the variables that depend on `component-active-bg`.